{
    "has_execution_error": true,
    "execution_output": "Fatal Python error: init_sys_streams: can't initialize sys standard streams\nPython runtime state: core initialized\nOSError: [Errno 9] Bad file descriptor\n\nCurrent thread 0x00000001dcac5c40 (most recent call first):\n  <no Python frame>\n",
    "pred": {
        "error": "malformed_output"
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "Nancy",
                "start_time": "09:16",
                "end_time": "10:46"
            },
            {
                "action": "meet",
                "person": "Stephanie",
                "start_time": "10:59",
                "end_time": "12:14"
            },
            {
                "action": "meet",
                "person": "Elizabeth",
                "start_time": "12:33",
                "end_time": "13:33"
            },
            {
                "action": "meet",
                "person": "Robert",
                "start_time": "13:42",
                "end_time": "14:27"
            },
            {
                "action": "meet",
                "person": "Melissa",
                "start_time": "14:48",
                "end_time": "15:18"
            },
            {
                "action": "meet",
                "person": "James",
                "start_time": "15:25",
                "end_time": "17:25"
            },
            {
                "action": "meet",
                "person": "Sarah",
                "start_time": "17:37",
                "end_time": "18:52"
            },
            {
                "action": "meet",
                "person": "Steven",
                "start_time": "19:15",
                "end_time": "19:30"
            }
        ]
    },
    "status": "Execution error: malformed_output",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 1
}